apps/backend/test/UploadTimesheet.ts

```diff
@@ -11,7 +11,7 @@ Utils file used in testing to upload entire timesheets
 
 
 const TIMEZONE = "America/New_York"; 
-const UUID = "4c8c5ad4-a8ab-4c92-b33f-b8f932b9e0b5"
```

Comment:
if possible can we throw this into the env so each dev can put in their own uuid?

Reply:
oh that's smart - @aditya161216 I can create a separate smaller ticket for this (or @VictoriaArzu can, if you want) since it's outside the scope for this

Reply:
@izzyconner sure another ticket for this would be great